native/src/rendering.cpp

```diff
@@ -1478,6 +1476,9 @@ void doRendering(std::vector<FoundMapDataObject>& mapDataObjects, SkCanvas* canv
 		updateTextTile(renderableObjects, rc);
 	}
 
+	for (auto& pair : renderableObjects) {
+		delete pair.second;
```

Comment:
Can be crash here. 
According code RenderableObject* can add more one time.
Also not all RenderableObject* can be added to renderableObjects (memory leak)

Reply:
clearTextTileObjectsCache